packages/suite/src/views/wallet/transactions/components/TransactionList/components/Actions/components/SearchAction/index.tsx

```diff
@@ -10,6 +10,7 @@ import { Account } from '@wallet-types';
 import { Translation } from '@suite-components';
 import { isEnabled } from '@suite-utils/features';
 import { useHotkeys } from 'react-hotkeys-hook';
+import { KEYBOARD_CODE } from '@trezor/components/src/constants/keyboardEvents';
```

Comment:
reexport `KEYBOARD_CODE` in `packages/components/src/index.ts` file

Reply:
Doing it! Thanks for the comment!